Wrappers/Python/cil/optimisation/functions/SubsetGradientAlgorithmFunction.py

```diff
@@ -0,0 +1,122 @@
+from cil.optimisation.functions import Function, SumFunction
+import numpy as np
+
+class SubsetGradientAlgorithmFunction(SumFunction):
```

Comment:
This should be renamed to `SubsetSumFunction`

Reply:
done in 41fb62aa3f5bc835b4651aa562d809081ff1df64